abd-clam/src/core/cluster/_cluster.rs

```diff
@@ -15,8 +15,8 @@ use std::fmt::Write;
 use crate::{utils, Dataset, Instance, PartitionCriteria, PartitionCriterion};
 
 /// Ratios are used for anomaly detection and related applications.
-pub type Ratios = [f64; 6];
-
+// pub type Ratios = [f64; 6];
+use crate::core::cluster::Ratios;
```

Comment:
Why did you move where this is defined? I prefer this here but if there's a good reason to move it, that's fine.

Reply:
I had trouble importing the Ratios type into other files. In pretrained_models for example, I could only get the 'use' statement to work when the type alias was in the mod file.

Reply:
In that case, remove the commented out lines of code